dashinterface.py

```diff
@@ -0,0 +1,1025 @@
+from urllib.parse import urlencode, quote, parse_qs
+from typing import Any, Dict, List, Tuple
+
+import dash
+import dash_bootstrap_components as dbc
+import dash_core_components as dcc
+import dash_html_components as html
+import dash_table
+import requests
+from dash.dependencies import Input, Output, State
+
+import tasks
+import views
+from app import app
+
+
+_example_usi = "mzspec:MSV000082796:KP_108_Positive:scan:1974"
```

Comment:
This spectrum doesn't look super great (the small peak at 1500 m/z). Is this a special spectrum? Can we find a more interesting default USI?